shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLTCLStatementSQLVisitor.java

```diff
@@ -94,7 +94,10 @@ public ASTNode visitRollback(final RollbackContext ctx) {
     
     @Override
     public ASTNode visitSavepoint(final SavepointContext ctx) {
-        return new MySQLSavepointStatement();
+        final String savepointName = ctx.getChild(1).getText();
```

Comment:
@roodkcab Please remove final for local param.

Reply:
no problem